v3/common/value_encoder_test.go

```diff
@@ -173,6 +173,76 @@ func TestValueEncoder_encodeLongs__Cycle(t *testing.T) {
 	}
 }
 
+func TestValueEncoder_encodeFloats__Cycle(t *testing.T) {
+	byteOrder := TestDefaultByteOrder
+	ve := NewValueEncoder(byteOrder)
+
+	original := []float32{17, 34, 51, 68, 85}
+
+	ed, err := ve.encodeFloats(original)
+	log.PanicIf(err)
+
+	if ed.Type != TypeFloat {
+		t.Fatalf("IFD type not expected.")
+	}
+
+	expected := []byte{
+		0x41, 0x88, 0x00, 0x00,
+		0x42, 0x08, 0x00, 0x00,
+		0x42, 0x4c, 0x00, 0x00,
+		0x42, 0x88, 0x00, 0x00,
+		0x42, 0xaa, 0x00, 0x00,
+	}
+
+	if reflect.DeepEqual(ed.Encoded, expected) != true {
```

Comment:
Can use `bytes.Equal()`. I didn't know about it when I originally wrote these tests.